### PR TITLE
Yahoo search implemented

### PR DESCRIPTION
**Pull request by abhishek1995s**
Added function to get yahoo search result and to scrape the text
Added button for yahoo search in front end

### CODE REVIEW

File: rss-generator.py [hunk 1]
```diff
@@ -27,6 +29,9 @@ def generateFeed(urls, stype):
         fe.title(url[0])
         fe.link({'href': url[1], 'rel': 'alternate'})
     print(fg.rss_str(pretty=True))
+    ##Write to file
+    file_name = 'data/%s.xml'%query
```
Comment (enigmaeth) [inline rss-generator.py]: Why is this file_name being used here?

File: rss-generator.py [hunk 2]
```diff
@@ -27,6 +29,9 @@ def generateFeed(urls, stype):
...
+    fg.rss_file(file_name)
```
Comment (enigmaeth) [inline rss-generator.py]: And here too.

File: rss-generator.py (hunk 1 above)
Comment (abhishek1995s) [inline rss-generator.py]: It was part of the code  to create xml file . I made no change that part

**Comment by abhishek1995s** [comment]
@niranjan94 I had problem in updating the previous pull requests. So I closed it and created a pull request with the changes

File: rss-generator.py (hunk 1 above)
Comment (abhishek1995s) [inline rss-generator.py]: Sorry I didn't update changes to my local repository will change it now

**Comment by abhishek1995s** [comment]
@enigmaeth I have made the changes to the code and now its the current one

File: rss-generator.py [hunk 3]
```diff
@@ -123,7 +124,35 @@ def google_search(query):
...
+    "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_8_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/27.0.1453.116 Safari/537.36"
+    }
+
+    payload = {'q' : query}
+    response = requests.get('https://search.yahoo.com/search', headers=header, params=payload)
 
+    return response
+
+def yahoo_search(query):
+    ''' Search google for the query and return set of urls
```
Comment (niranjan94) [inline rss-generator.py]: Please update the comment correctly

**Comment by abhishek1995s** [comment]
@niranjan94 I have updated the comments

**Comment by niranjan94** [comment]
@abhishek1995s please refrain from closing and opening PRs ... makes it hard to keep track of conversations in PR. Just force push to the same branch and your PR will get automatically updated.

**Pull request closed without merging**